content/services/csv/powerdev_open_source_projects.csv

```diff
@@ -35,6 +35,7 @@
 "glibc","The GNU C Library project provides the core libraries for the GNU system and GNU/Linux systems"
 "Go Language","Supports project specific porting efforts of POWER8 ppc64/ppc64le"
 "Hadoop_Spark","CI environment powered by a Jenkins server running Hadoop and Spark builds for including POWER into the Hyperledger fabric community to perform Continuous Integration for IBM Hyperledger fabric codebase."
+"Heptapod","Project adding Mercurial support to GitLab"
```

Comment:
This shouldn't be listed on the powerdev projects. Instead it should be on the general OSL projects page.